internal/connection/connection.go

```diff
@@ -29,23 +31,60 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const (
+	// The duration after which a gRPC connection is closed due to inactivity
+	IDLE_TIMEOUT = time.Minute * 5
```

Comment:
I guess `defaultIdleTimeout` would more suitable, and maybe have it as an optional configuration option in the configmap?